backend/pkg/api/connect/service/console/stream_progress_reporter.go

```diff
@@ -37,17 +37,33 @@ type streamProgressReporter struct {
 }
 
 func (p *streamProgressReporter) Start() {
-	// If search is disabled do not report progress regularly as each consumed message will be sent through the socket
-	// anyways
-	if p.request.FilterInterpreterCode == "" {
+	tickerDuration := 0 * time.Second
+
+	// We should report progress in two scenarios.
+	// If filter is enabled it could take a while to find the matching record(s).
+	// If search is for newest records it could take a while to get new records.
+	// For the second scenario we also need to get around certain infrastructure and ingress idle connection limits.
```

Comment:
> For the second scenario we also need to get around certain infrastructure

That's true for both cases as both can take a while until we actually send messages via the stream. The difference was that we already used to send a progress report (how many messages & bytes we have consumed) when we use pushdown filters.

Reply:
Improved comments.